tools/snippet-testing/snippet_tester.rb

```diff
@@ -230,9 +244,11 @@ def parse_options(args)
     tester = SnippetTester.new(options.source_folder, options.test_default)
 
     tester.install_dependencies if options.install
+    tester.add_hosts if options.update_hosts
     tester.init
     tester.setup
     tester.run
+    tester.remove_hosts if options.update_hosts
```

Comment:
If anything stops the test runner before it finishes, we end up with a `/etc/hosts` modified that will affect any request made to Twilio's API.